document/core/util/mathjax2katex.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
```

Comment:
should this be python3 now?

Reply:
Think this will be python3 since the travis setup only has python3. But will be useful for local builds. I will make the change.